release_util/tests/migrations/test_migrations/0004_fourth.py

```diff
@@ -0,0 +1,25 @@
+from django.db import migrations, models
```

Comment:
These migrations are not of this ticket. Make them against master branch in separate PR.